.copilot-instructions

```diff
@@ -0,0 +1,134 @@
+- **Theme**: next-themes for dark/light mode support
+
+## File Organization
+
+### Backend Structure
+```
+/cmd/main.go                    # Application entry point
+/internal/
+  ├── handlers/                 # HTTP request handlers
+  ├── middlewares/              # Echo middlewares (auth, CORS, logging)
+  ├── routers/                  # Route definitions
+  ├── services/                 # Business logic and external service clients
+  └── client/                   # gRPC client implementations
+/configs/                       # Configuration structs and loading
+/api/                          # Generated OpenAPI specs
+```
+
+### Frontend Structure
+```
+/website/src/
+  ├── components/               # Reusable UI components
+  │   ├── ui/                  # Base UI components (shadcn/ui style)
+  │   └── AdminApp.tsx         # Main application component
+  ├── pages/                   # Page components
+  ├── hooks/                   # Custom React hooks
+  ├── lib/                     # Utility functions
+  ├── i18n/                    # Internationalization files
+  └── api/                     # Generated TypeScript API clients
+```
+
+## Development Workflow
+
+### API Development
+1. Add Swagger annotations to Go handlers using `@Summary`, `@Description`, `@Tags`, etc.
+2. Run `make swag` to generate OpenAPI spec and TypeScript clients
+3. Use generated TypeScript types and API functions in frontend
+
+### Building and Running
+- **Backend**: `make build` creates binary in `/bin/web`
+- **Frontend**: `cd website && pnpm run build`
+- **Development**: Use `make` targets and VS Code launch configurations
+
+## Coding Conventions
+
+### Go
+- Use dependency injection via service manager pattern
+- Handler functions should follow RESTful conventions
+- Error handling with proper HTTP status codes and gRPC error mapping
+- Use contexts for request tracing and cancellation
+- Follow Go naming conventions (PascalCase for exports, camelCase for private)
+
+### TypeScript/React
+- Use functional components with hooks
+- Prop destructuring in component parameters
+- TypeScript strict mode enabled
+- Component composition over inheritance
+- Custom hooks for business logic
+- Consistent import ordering: external libraries, internal modules, relative imports
+
+### API Patterns
+- RESTful endpoints under `/api/v1/`
+- JWT authentication via Bearer tokens
+- gRPC metadata for service-to-service auth
+- Standardized error responses with HTTP status codes
+
+## Technology Stack
+
+### Backend Dependencies
+- `github.com/labstack/echo/v4` - HTTP framework
+- `github.com/oj-lab/go-webmods` - Web application utilities
+- `github.com/oj-lab/user-service` - User authentication service
+- `google.golang.org/grpc` - gRPC communication
+- `github.com/spf13/viper` - Configuration management
+
+### Frontend Dependencies
+- `react` & `react-dom` - UI framework
+- `@radix-ui/*` - Accessible UI primitives
```

Comment:
Do not use radix-ui, use Shadcn. If project have radix-ui code currently remove them.